packages/cli/src/databases/migrations/sqlite/1724951148974-AddApiKeysTable.ts

```diff
@@ -74,4 +74,52 @@ export class AddApiKeysTable1724951148974 {
 		// Rename the temporary table to users
 		await queryRunner.query('ALTER TABLE users_new RENAME TO user;');
 	}
+
+	async down({
+		queryRunner,
+		runQuery,
+		schemaBuilder: { dropTable, addColumns, createIndex, column },
+		escape,
+	}: MigrationContext) {
+		const userTable = escape.tableName('user');
+		const userApiKeysTable = escape.tableName('user_api_keys');
+		const apiKeyColumn = escape.columnName('apiKey');
+		const userIdColumn = escape.columnName('userId');
+		const idColumn = escape.columnName('id');
+		const createdAtColumn = escape.columnName('createdAt');
+
+		const queryToGetUsersApiKeys = `
+		SELECT
+			${userIdColumn},
+			${apiKeyColumn},
+			${createdAtColumn}
+	FROM
+		${userApiKeysTable}
+	WHERE
+		${createdAtColumn} IN(
+			SELECT
+				MIN(${createdAtColumn})
+				FROM ${userApiKeysTable}
+			GROUP BY ${userIdColumn});`;
+
+		const oldestApiKeysPerUser = (await queryRunner.query(queryToGetUsersApiKeys)) as Array<
```

Comment:
For some strange reason running a query agains the `user_api_keys` before the `addColumns` always return empty, so had to query the API keys before.

Reply:
this was probably happening because of the truncation. we could probably move this to before `addColumns` if needed. but we don't need to block this PR on that change.